4 - Tables and Persistence/7 - Saving Data/4.7 Lab/EmojiDictionary/EmojiDictionary/Emoji.swift

```diff
@@ -0,0 +1,45 @@
+
+import Foundation
+
+
+// make struct conform to Codable
+struct Emoji: Codable {
+    var symbol: String
+    var name: String
+    var detailDescription: String
+    var usage: String
+    static var ArchiveURL = documentsDirectory.appendingPathComponent("emojis").appendingPathExtension("plist")
+
+    //implement saving and loading methods
+    static func saveToFile(emojis: [Emoji]) {
```

Comment:
@Natasha-A  good job on these methods. Where are you calling these methods? You will have to call them where it makes sense in EmojiTableViewController so that you can retrieve/save the data